docs/content/dagster-cloud/developing-testing/branch-deployments/using-branch-deployments-with-github.mdx

```diff
@@ -0,0 +1,302 @@
+---
+title: Using Branch Deployments with GitHub Actions | Dagster Cloud
+description: Develop and test in the cloud.
+---
+
+# Using Branch Deployments with GitHub Actions
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+In this guide, we'll walk you through setting up [Branch Deployments](/dagster-cloud/developing-testing/branch-deployments) with GitHub Actions.
+
+**If you don't use GitHub for version control**, check out the dedicated [Branch deployments with TODO guide](/dagster-cloud/developing-testing/branch-deployments/using-branch-deployments).
+
+---
+
+## Prerequisites
+
+Utilizing Branch Deployments requires setting up two components: the Branch Deployment agent and continuous integration (CI) with GitHub Actions. You will need:
+
+- **Editor** or **Admin** permissions in Dagster Cloud
+- The ability to run a new agent in your infrastructure
```

Comment:
Unsure how we are handling this elsewhere in the docs, but we may need to have separate instructions for Serverless vs Hybrid. Serverless, no agent setup is needed here.

The Hybrid case will depend on whether we recommend running the same agent for traditional & branch deployments (in which case this step becomes "the ability to change your agent's configuration") or a new agent (in which case this step remains the same). cc @gibsondan

Reply:
Noted. I'm going to merge this for now so it can be surfaced publicly for feedback, and open another PR to address this.